docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc

```diff
@@ -92,6 +92,27 @@ To disable this behavior, use the SPI option `spi-single-use-object-infinispan-p
 The SPI behavior of `SingleUseObjectProvider` has changed that for revoked tokens only the methods `put` and `contains` must be used.
 This is enforced by default, and can be disabled using the SPI option `spi-single-use-object-infinispan-persist-revoked-tokens`.
 
+= HA Multi-Site Deployments
+
+{project_name} 26 introduces significant improvements to the recommended HA multi-site architecture, most notably:
+
+- {project_name} deployments on each site are now able to handle user requests simulaneously, Active/Active vs the previous
+Active/Passive architecture.
```

Comment:
```suggestion
- {project_name} deployments on each site are now able to handle user requests simultaneously, therefore active/active setups are supported in additional to the previous active/passive setups.
```

Reply:
Do we want to go as far as saying that the previous active/passive setup is supported given that we have removed all docs? I thought we would allow users to continue with that configuration, but not encourage it ... maybe I'm reading too much into the word "supported".

Reply:
I agree that supported is a big word here. What about: 

> ... therefore active/active setups are now supported, while previous configurations which leveraged active/passive loadbalancers will continue to work.